defaults-o2-dev-fairroot.sh

```diff
@@ -5,8 +5,6 @@ env:
   CFLAGS: "-fPIC -g -O2"
   CMAKE_BUILD_TYPE: "RELWITHDEBINFO"
 disable:
-  - AliEn-Runtime
-  - AliRoot
```

Comment:
No no ... removing this line just does not prohibit building AliRoot with this default. It does not make AliRoot a dependency (at least with #817).